marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/lib/entitySearchService.sjs

```diff
@@ -1,5 +1,11 @@
+const hubTest = require("/test/data-hub-test-helper.sjs");
+
 function invokeSavedQuery(module, args) {
-  return fn.head(xdmp.invoke("/data-hub/5/data-services/entitySearch/" + module, args));
+  return fn.head(hubTest.runWithRolesAndPrivileges(
+      ['data-hub-developer', 'hub-central-saved-query-reader', 'hub-central-saved-query-writer'], [],
```

Comment:
Shouldn't you be able to do this without the "data-hub-developer" role, since this is an HC-specific capability?

Reply:
Yes. It should. The any-uri, any-collection privileges should be added to hub-central-saved-query-writer.

Reply:
I think hub-central-saved-query-writer just needs to inherit data-hub-common then, right @ryanjdew ?

Reply:
data-hub-common-writer has the any-uri and unprotected-collections privileges. Should I extend that role?

Reply:
Yes, I think that's the way to go.